conda/build.sh

```diff
@@ -1,21 +1,65 @@
 #!/bin/bash
 
-set -e
+# Install script for Anaconda environments on macOS. This will only work on mac
+# This script is not supposed to be called correctly, but should be run by:
+#
+# $ cd <path to caffe2, e.g. ~/caffe2>
+# $ conda build conda/build
+#
+# This installation uses MKL and does not use CUDA
+#
+# If you're debugging this, it may be useful to use the env that conda build is
+# using:
+# $ cd <anaconda_root>/conda-bld/caffe2_<timestamp>
+# $ source activate _h_env_... # some long path with lots of placeholders
+#
+# Also, failed builds will accumulate those caffe2_<timestamp> directories. You
+# can remove them after a succesfull build with
+# $ conda build purge
+
+set -ex
 
 if [ -z "$PREFIX" ]; then
   PREFIX="$CONDA_PREFIX"
 fi
+echo "Installing caffe2 to ${PREFIX}"
 
 # conda build will copy everything over, including build directories.
-# Don't let this pollute hte build!
+# Don't let this pollute the build!
 rm -rf build || true
 
+# Default leveldb from conda-forge doesn't work
+pip install leveldb==0.18
```

Comment:
Does this mean that a default conda install of the package won't work?

Reply:
@pietern We can just turn leveldb off. I will remove this, as we do turn leveldb off later.

@harouwu Sorry I didn't notice your question earlier. These are the first steps to supporting conda install, but this PR won't enable that just yet